packages/contract-wrappers/src/contract_wrappers.ts

```diff
@@ -69,6 +74,8 @@ export class ContractWrappers {
      */
     public coordinator: CoordinatorWrapper;
 
+    public constants: { [key: string]: any } = wrapperConstants;
```

Comment:
Let's not do this. There was one constant needed in a single file, let's just add the constant to that package instead.